### PR TITLE
Set resources for multiple addons

### PR DESCRIPTION
**Pull request by dkoshkin**
Blocked by: https://github.com/mesosphere/charts/pull/97

### CODE REVIEW

File: templates/metallb.yaml [hunk 1]
```diff
@@ -21,4 +21,11 @@ spec:
     version: 0.9.5
     values: |
       ---
-      # we currently provide no additional configuration for metallb
+      controller:
+        limits:
+          cpu: 100m
+          memory: 100Mi
+      speaker:
+        limits:
```
Comment (jieyu) [inline templates/metallb.yaml]: set requests instead?

File: templates/metallb.yaml [hunk 2]
```diff
@@ -21,4 +21,11 @@ spec:
     version: 0.9.5
     values: |
       ---
-      # we currently provide no additional configuration for metallb
+      controller:
+        limits:
```
Comment (jieyu) [inline templates/metallb.yaml]: set requests instead?

File: templates/metallb.yaml (hunk 1 above)
Comment (hectorj2f) [reply templates/metallb.yaml]: good call

File: templates/metallb.yaml (hunk 2 above)
Comment (hectorj2f) [reply templates/metallb.yaml]: yep

**Comment by hectorj2f** [comment]
I am going to merge this to start testing it

**Comment by hectorj2f** [comment]
okay... i am signing the commits

**Pull request merged**